core/src/main/java/aima/core/util/math/MixedRadixInterval.java

```diff
@@ -110,6 +110,38 @@ public BigInteger getValueFor(int[] numeralValues) {
 		return cvalue;
 	}
 
+	/**
+	 * Returns the value of the mixed radix interval by ignoring radix indices
+	 * specified in leaveOutSubset given the specified array of numerals.
+	 *
+	 * @param numeralValues[]
+	 *            is the array of values for each radix position.
+	 * @param leaveOutSubset[]
+	 *            is the array of indices of radix positions which are ignored
+	 *            when computing the mixed interval value.
+	 *
+	 * @return the value of the numerals at within the interval.
+	 */
+	public BigInteger subsetInterval(int numeralValues[], int leaveOutSubset[]) {
+		Arrays.sort(leaveOutSubset);
+		BigInteger cvalue = BigInteger.ZERO;
+		BigInteger mvalue = BigInteger.ONE;
+		int c = -1, j = leaveOutSubset.length - 1;
+		for (int i = numeralValues.length - 1; i >= 0; i--) {
```

Comment:
c and j index values are not updated in following logic.
can you add test cases for this method so it will be easier to follow its functionality.